server/models/User.js

```diff
@@ -16,23 +17,25 @@ const userSchema = new mongoose.Schema({
     type: String,
     required: true
   },
+
   register_date: {
     type: Date,
     default: Date.now
-  }
-});
+  },
+  profile: { type: Schema.Types.Object, ref: 'profile' }
```

Comment:
Wouldn't this be `Schema.Types.ObjectId` here? :)

Reply:
I have tried using "Schema.Types.ObjectId" before but it only generate the ObjectKey without adding the property,that is why I used  "Schema.Types.Object" and it worked